### ISSUE TITLE
flask 2 support

### ISSUE THREAD

**Issue by tbhi** [spulec/moto#3945]
Trying to install moto[server] and it seems that the recent flask 2 release might be causing problems

```
There are incompatible versions in the resolved dependencies:
  werkzeug<2.0.0 (from moto[server]==2.0.7->-r /tmp/pipenvwdej_qnurequirements/pipenv-u03fqotz-constraints.txt (line 3))
  werkzeug>=2.0 (from flask==2.0.0->moto[server]==2.0.7->-r /tmp/pipenvwdej_qnurequirements/pipenv-u03fqotz-constraints.txt (line 3))
```
Adding `flask = '<2'` to my Pipfile resolves the issue.

**Comment by bblommers** [comment]
Correct @tbhi - moto 2.0.7 doesn't work with flask 2.x. This will be fixed in the next release, 2.0.8. (If you can't wait, moto 2.0.6 should work as well.)

**Comment by bblommers** [comment]
Moto 2.0.8 has just been released, and doesn't have any dependency constraints on flask/werkzeug anymore 👍
